packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -79,6 +79,25 @@ Given(
   }
 );
 
+Given(
+  'I intercept {string} with fixture {string} and add header {string} with value {string}',
```

Comment:
Unless I'm reading wrong think we can probably hard code the second and third positional parameters here since all call sites ate passing the same values. IMO unlikely that the function will be reused in the future, and can add the parameters if/when needed